### ISSUE TITLE
Audio Control Module: Buffer Overrun (8-x-y)

### ISSUE THREAD

**Issue by chevonc** [electron/electron#25492]
### Preflight Checklist

* [ X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ X] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 8.4.1
* **Operating System:**
  * Windows 10 x64
* **Last Known Working Electron version:**
  * 7.4.1

### Expected Behavior
Audio devices with high sample rates don't cause buffer overflow in native `audio_control_module.cc`. This issue was fixed upstream 6months ago but should have been backported to electron 8-x-y because the Chromium ver. used doesn't include the fix yet. 

Electron 7-x-y backport: https://github.com/electron/electron/pull/23038/files
Chrome's Fix: https://webrtc.googlesource.com/src/+/d82a02c837d33cdfd75121e40dcccd32515e42d6 to  

Expected patch for backported fix, but it's not there.

### Actual Behavior
If windows audio devices are set to >48kHz sample rates, buffer overrun occurs in `audio_control_module.cc` and causes renderer to crash. I believe this is because electron 8-x-y is pinned to chrome `80.0.3987.165`, which has a webrtc DEPS at hash `7a0e44c1a84fb4ed57a6701cfc8093756c37af6f` from [Jan 17](https://chromium.googlesource.com/external/webrtc/+/7a0e44c1a84fb4ed57a6701cfc8093756c37af6f) that doesn't include the acm fix.

App Crashes like this
![image](https://user-images.githubusercontent.com/684657/93369723-a92bd080-f804-11ea-8e7e-e5c3ec597cd8.png)

### To Reproduce
Run JS audio script processor on audio track received from `getUserMedia`. When attempting to add 10MS of audio data, the renderer crashes here: 

### Screenshots
**Exception that occurs**
![image](https://user-images.githubusercontent.com/684657/93369489-4d614780-f804-11ea-8e46-1189446fbca0.png)

**8-x-y Chrome Ver**
![image](https://user-images.githubusercontent.com/684657/93369549-62d67180-f804-11ea-93ee-794744ae822f.png)

### Additional Information

<details>
<summary>Stack Error Info</summary>
00007FF66E5F9A74  xor         rax,rsp  
00007FF66E5F9A77  mov         qword ptr [rsp+28h],rax  
00007FF66E5F9A7C  lea         rdx,[rcx+3E48h]  
00007FF66E5F9A83  lea         rbx,[lock]  
00007FF66E5F9A88  mov         rcx,rbx  
00007FF66E5F9A8B  call        rtc::CritScope::CritScope (07FF66BE3D8ACh)  
00007FF66E5F9A90  mov         qword ptr [rdi+3E70h],rsi  
00007FF66E5F9A97  mov         rcx,rbx  
00007FF66E5F9A9A  call        rtc::CritScope::~CritScope (07FF66BE3D8CAh)  
00007FF66E5F9A9F  mov         rcx,qword ptr [rsp+28h]  
00007FF66E5F9AA4  xor         rcx,rsp  
00007FF66E5F9AA7  call        __security_check_cookie (07FF66E6BF1A0h)  
00007FF66E5F9AAC  xor         eax,eax  
00007FF66E5F9AAE  add         rsp,30h  
00007FF66E5F9AB2  pop         rbx  
00007FF66E5F9AB3  pop         rdi  
00007FF66E5F9AB4  pop         rsi  
00007FF66E5F9AB5  ret  
--- No source file -------------------------------------------------------------
00007FF66E5F9AB6  int         3  
00007FF66E5F9AB7  int         3  
00007FF66E5F9AB8  int         3  
00007FF66E5F9AB9  int         3  
00007FF66E5F9ABA  int         3  
00007FF66E5F9ABB  int         3  
00007FF66E5F9ABC  int         3  
00007FF66E5F9ABD  int         3  
00007FF66E5F9ABE  int         3  
00007FF66E5F9ABF  int         3  
--- C:\projects\src\third_party\webrtc\modules\audio_coding\acm2\audio_coding_module.cc 
00007FF66E5F9AC0  push        r15  
00007FF66E5F9AC2  push        r14  
00007FF66E5F9AC4  push        r13  
00007FF66E5F9AC6  push        r12  
00007FF66E5F9AC8  push        rsi  
00007FF66E5F9AC9  push        rdi  
00007FF66E5F9ACA  push        rbp  
00007FF66E5F9ACB  push        rbx  
00007FF66E5F9ACC  sub         rsp,918h  
00007FF66E5F9AD3  mov         rdi,rdx  
00007FF66E5F9AD6  mov         rsi,rcx  
00007FF66E5F9AD9  mov         rax,qword ptr [__security_cookie (07FF66F922B28h)]  
00007FF66E5F9AE0  xor         rax,rsp  
00007FF66E5F9AE3  mov         qword ptr [rsp+910h],rax  
00007FF66E5F9AEB  lea         rdx,[rcx+40h]  
00007FF66E5F9AEF  lea         rcx,[lock]  
00007FF66E5F9AF4  call        rtc::CritScope::CritScope (07FF66BE3D8ACh)  
00007FF66E5F9AF9  mov         rax,qword ptr [rdi+18h]  
00007FF66E5F9AFD  test        rax,rax  
00007FF66E5F9B00  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+98h (07FF66E5F9B58h)  
00007FF66E5F9B02  mov         ecx,dword ptr [rdi+20h]  
00007FF66E5F9B05  cmp         ecx,2EE01h  
00007FF66E5F9B0B  jl          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+0DFh (07FF66E5F9B9Fh)  
00007FF66E5F9B11  xor         eax,eax  
00007FF66E5F9B13  mov         dword ptr [rsp+28h],eax  
00007FF66E5F9B17  mov         dword ptr [rsp+20h],eax  
00007FF66E5F9B1B  lea         rdx,[string "../../third_party/webrtc/modules"... (07FF66F7DDCE1h)]  
00007FF66E5F9B22  lea         rsi,[rsp+190h]  
00007FF66E5F9B2A  mov         rcx,rsi  
00007FF66E5F9B2D  mov         r8d,161h  
00007FF66E5F9B33  mov         r9d,1  
00007FF66E5F9B39  call        rtc::DiagnosticLogMessage::DiagnosticLogMessage (07FF66B83A4A4h)  
00007FF66E5F9B3E  lea         rcx,[rsp+1B8h]  
00007FF66E5F9B46  lea         rdx,[string "Cannot Add 10 ms audio, input fr"... (07FF66F7DDD5Ah)]  
00007FF66E5F9B4D  mov         r8d,31h  
00007FF66E5F9B53  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+1B2h (07FF66E5F9C72h)  
00007FF66E5F9B58  xor         eax,eax  
00007FF66E5F9B5A  mov         dword ptr [rsp+28h],eax  
00007FF66E5F9B5E  mov         dword ptr [rsp+20h],eax  
00007FF66E5F9B62  lea         rdx,[string "../../third_party/webrtc/modules"... (07FF66F7DDCE1h)]  
00007FF66E5F9B69  lea         rsi,[rsp+190h]  
00007FF66E5F9B71  mov         rcx,rsi  
00007FF66E5F9B74  mov         r8d,15Bh  
00007FF66E5F9B7A  mov         r9d,1  
00007FF66E5F9B80  call        rtc::DiagnosticLogMessage::DiagnosticLogMessage (07FF66B83A4A4h)  
00007FF66E5F9B85  lea         rcx,[rsp+1B8h]  
00007FF66E5F9B8D  lea         rdx,[string "Cannot Add 10 ms audio, payload "... (07FF66F7DDD2Bh)]  
00007FF66E5F9B94  mov         r8d,2Eh  
00007FF66E5F9B9A  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+1B2h (07FF66E5F9C72h)  
00007FF66E5F9B9F  movsxd      rcx,ecx  
00007FF66E5F9BA2  imul        rcx,rcx,51EB851Fh  
00007FF66E5F9BA9  mov         rdx,rcx  
00007FF66E5F9BAC  shr         rdx,3Fh  
00007FF66E5F9BB0  sar         rcx,25h  
00007FF66E5F9BB4  add         ecx,edx  
00007FF66E5F9BB6  movsxd      rcx,ecx  
00007FF66E5F9BB9  cmp         rax,rcx  
00007FF66E5F9BBC  jne         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+170h (07FF66E5F9C30h)  
00007FF66E5F9BBE  mov         rax,qword ptr [rdi+28h]  
00007FF66E5F9BC2  cmp         rax,8  
00007FF66E5F9BC6  ja          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+2C8h (07FF66E5F9D88h)  
00007FF66E5F9BCC  mov         ecx,156h  
00007FF66E5F9BD1  bt          rcx,rax  
00007FF66E5F9BD5  jae         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+2C8h (07FF66E5F9D88h)  
00007FF66E5F9BDB  lea         rdx,[string "Add10MsData" (07FF66F7DDE01h)]  
00007FF66E5F9BE2  mov         rcx,rsi  
00007FF66E5F9BE5  call        webrtc::`anonymous namespace'::AudioCodingModuleImpl::HaveValidEncoder (07FF66E5FA69Ch)  
00007FF66E5F9BEA  mov         ebx,0FFFFFFFFh  
00007FF66E5F9BEF  test        al,al  
00007FF66E5F9BF1  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+1C4h (07FF66E5F9C84h)  
00007FF66E5F9BF7  mov         r12d,dword ptr [rdi+20h]  
00007FF66E5F9BFB  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9C02  mov         rax,qword ptr [rcx]  
00007FF66E5F9C05  call        qword ptr [rax+8]  
00007FF66E5F9C08  mov         r14d,eax  
00007FF66E5F9C0B  cmp         qword ptr [rdi+28h],2  
00007FF66E5F9C10  jne         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+1F4h (07FF66E5F9CB4h)  
00007FF66E5F9C16  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9C1D  mov         rax,qword ptr [rcx]  
00007FF66E5F9C20  call        qword ptr [rax+10h]  
00007FF66E5F9C23  cmp         rax,1  
00007FF66E5F9C27  sete        r13b  
00007FF66E5F9C2B  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+1F7h (07FF66E5F9CB7h)  
00007FF66E5F9C30  xor         eax,eax  
00007FF66E5F9C32  mov         dword ptr [rsp+28h],eax  
00007FF66E5F9C36  mov         dword ptr [rsp+20h],eax  
00007FF66E5F9C3A  lea         rdx,[string "../../third_party/webrtc/modules"... (07FF66F7DDCE1h)]  
00007FF66E5F9C41  lea         rsi,[rsp+190h]  
00007FF66E5F9C49  mov         rcx,rsi  
00007FF66E5F9C4C  mov         r8d,168h  
00007FF66E5F9C52  mov         r9d,1  
00007FF66E5F9C58  call        rtc::DiagnosticLogMessage::DiagnosticLogMessage (07FF66B83A4A4h)  
00007FF66E5F9C5D  lea         rcx,[rsp+1B8h]  
00007FF66E5F9C65  lea         rdx,[string "Cannot Add 10 ms audio, input fr"... (07FF66F7DDD8Ch)]  
00007FF66E5F9C6C  mov         r8d,40h  
00007FF66E5F9C72  call        std::__1::__put_character_sequence<char,std::__1::char_traits<char> > (07FF6699A21D3h)  
00007FF66E5F9C77  mov         rcx,rsi  
00007FF66E5F9C7A  call        rtc::DiagnosticLogMessage::~DiagnosticLogMessage (07FF66B83A5DEh)  
00007FF66E5F9C7F  mov         ebx,0FFFFFFFFh  
00007FF66E5F9C84  lea         rcx,[lock]  
00007FF66E5F9C89  call        rtc::CritScope::~CritScope (07FF66BE3D8CAh)  
00007FF66E5F9C8E  mov         rcx,qword ptr [rsp+910h]  
00007FF66E5F9C96  xor         rcx,rsp  
00007FF66E5F9C99  call        __security_check_cookie (07FF66E6BF1A0h)  
00007FF66E5F9C9E  mov         eax,ebx  
00007FF66E5F9CA0  add         rsp,918h  
00007FF66E5F9CA7  pop         rbx  
00007FF66E5F9CA8  pop         rbp  
00007FF66E5F9CA9  pop         rdi  
00007FF66E5F9CAA  pop         rsi  
00007FF66E5F9CAB  pop         r12  
00007FF66E5F9CAD  pop         r13  
00007FF66E5F9CAF  pop         r14  
00007FF66E5F9CB1  pop         r15  
00007FF66E5F9CB3  ret  
00007FF66E5F9CB4  xor         r13d,r13d  
00007FF66E5F9CB7  cmp         byte ptr [rsi+3E38h],0  
00007FF66E5F9CBE  mov         eax,dword ptr [rdi]  
00007FF66E5F9CC0  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+30Fh (07FF66E5F9DCFh)  
00007FF66E5F9CC6  cmp         eax,dword ptr [rsi+84h]  
00007FF66E5F9CCC  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+324h (07FF66E5F9DE4h)  
00007FF66E5F9CD2  xor         eax,eax  
00007FF66E5F9CD4  mov         dword ptr [rsp+28h],eax  
00007FF66E5F9CD8  mov         dword ptr [rsp+20h],eax  
00007FF66E5F9CDC  lea         rdx,[string "../../third_party/webrtc/modules"... (07FF66F7DDCE1h)]  
00007FF66E5F9CE3  lea         rbp,[rsp+190h]  
00007FF66E5F9CEB  mov         rcx,rbp  
00007FF66E5F9CEE  mov         r8d,1B5h  
00007FF66E5F9CF4  mov         r9d,2  
00007FF66E5F9CFA  call        rtc::DiagnosticLogMessage::DiagnosticLogMessage (07FF66B83A4A4h)  
00007FF66E5F9CFF  lea         rcx,[rsp+1B8h]  
00007FF66E5F9D07  lea         rdx,[string "Unexpected input timestamp: " (07FF66F7DDE33h)]  
00007FF66E5F9D0E  mov         r8d,1Ch  
00007FF66E5F9D14  call        std::__1::__put_character_sequence<char,std::__1::char_traits<char> > (07FF6699A21D3h)  
00007FF66E5F9D19  mov         edx,dword ptr [rdi]  
00007FF66E5F9D1B  mov         rcx,rax  
00007FF66E5F9D1E  call        std::__1::basic_ostream<char,std::__1::char_traits<char> >::operator<< (07FF66B66FC50h)  
00007FF66E5F9D23  lea         rdx,[string ", expected: " (07FF66E72E62Dh)]  
00007FF66E5F9D2A  mov         r8d,0Ch  
00007FF66E5F9D30  mov         rcx,rax  
00007FF66E5F9D33  call        std::__1::__put_character_sequence<char,std::__1::char_traits<char> > (07FF6699A21D3h)  
00007FF66E5F9D38  mov         edx,dword ptr [rsi+84h]  
00007FF66E5F9D3E  mov         rcx,rax  
00007FF66E5F9D41  call        std::__1::basic_ostream<char,std::__1::char_traits<char> >::operator<< (07FF66B66FC50h)  
00007FF66E5F9D46  mov         rcx,rbp  
00007FF66E5F9D49  call        rtc::DiagnosticLogMessage::~DiagnosticLogMessage (07FF66B83A5DEh)  
00007FF66E5F9D4E  mov         ebp,dword ptr [rdi]  
00007FF66E5F9D50  sub         ebp,dword ptr [rsi+84h]  
00007FF66E5F9D56  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9D5D  mov         rax,qword ptr [rcx]  
00007FF66E5F9D60  call        qword ptr [rax+8]  
00007FF66E5F9D63  cvtsi2sd    xmm0,eax  
00007FF66E5F9D67  cvtsi2sd    xmm1,dword ptr [rdi+20h]  
00007FF66E5F9D6C  divsd       xmm0,xmm1  
00007FF66E5F9D70  cvttsd2si   rax,xmm0  
00007FF66E5F9D75  imul        eax,ebp  
00007FF66E5F9D78  add         dword ptr [rsi+80h],eax  
00007FF66E5F9D7E  mov         eax,dword ptr [rdi]  
00007FF66E5F9D80  mov         dword ptr [rsi+84h],eax  
00007FF66E5F9D86  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+324h (07FF66E5F9DE4h)  
00007FF66E5F9D88  xor         eax,eax  
00007FF66E5F9D8A  mov         dword ptr [rsp+28h],eax  
00007FF66E5F9D8E  mov         dword ptr [rsp+20h],eax  
00007FF66E5F9D92  lea         rdx,[string "../../third_party/webrtc/modules"... (07FF66F7DDCE1h)]  
00007FF66E5F9D99  lea         rsi,[rsp+190h]  
00007FF66E5F9DA1  mov         rcx,rsi  
00007FF66E5F9DA4  mov         r8d,170h  
00007FF66E5F9DAA  mov         r9d,1  
00007FF66E5F9DB0  call        rtc::DiagnosticLogMessage::DiagnosticLogMessage (07FF66B83A4A4h)  
00007FF66E5F9DB5  lea         rcx,[rsp+1B8h]  
00007FF66E5F9DBD  lea         rdx,[string "Cannot Add 10 ms audio, invalid "... (07FF66F7DDDCDh)]  
00007FF66E5F9DC4  mov         r8d,33h  
00007FF66E5F9DCA  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+1B2h (07FF66E5F9C72h)  
00007FF66E5F9DCF  mov         dword ptr [rsi+84h],eax  
00007FF66E5F9DD5  mov         ecx,dword ptr [rdi]  
00007FF66E5F9DD7  mov         dword ptr [rsi+80h],ecx  
00007FF66E5F9DDD  mov         byte ptr [rsi+3E38h],1  
00007FF66E5F9DE4  cmp         r12d,r14d  
00007FF66E5F9DE7  setne       cl  
00007FF66E5F9DEA  or          cl,r13b  
00007FF66E5F9DED  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+368h (07FF66E5F9E28h)  
00007FF66E5F9DEF  lea         rbp,[rsi+1E0h]  
00007FF66E5F9DF6  mov         rax,qword ptr [rdi+28h]  
00007FF66E5F9DFA  mov         qword ptr [rsi+208h],rax  
00007FF66E5F9E01  mov         rax,qword ptr [rdi+18h]  
00007FF66E5F9E05  mov         qword ptr [rsi+1F8h],rax  
00007FF66E5F9E0C  mov         rcx,rdi  
00007FF66E5F9E0F  call        webrtc::AudioFrame::data (07FF66BE7FF20h)  
00007FF66E5F9E14  test        r13b,r13b  
00007FF66E5F9E17  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+3B0h (07FF66E5F9E70h)  
00007FF66E5F9E19  cmp         r12d,r14d  
00007FF66E5F9E1C  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+3B5h (07FF66E5F9E75h)  
00007FF66E5F9E1E  lea         rax,[rsp+190h]  
00007FF66E5F9E26  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+3BDh (07FF66E5F9E7Dh)  
00007FF66E5F9E28  mov         ecx,eax  
00007FF66E5F9E2A  mov         rbp,rdi  
00007FF66E5F9E2D  cmp         eax,dword ptr [rsi+80h]  
00007FF66E5F9E33  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+399h (07FF66E5F9E59h)  
00007FF66E5F9E35  lea         rbp,[rsi+1E0h]  
00007FF66E5F9E3C  mov         rcx,rbp  
00007FF66E5F9E3F  mov         rdx,rdi  
00007FF66E5F9E42  call        webrtc::AudioFrame::CopyFrom (07FF66BE7FE26h)  
00007FF66E5F9E47  mov         eax,dword ptr [rsi+80h]  
00007FF66E5F9E4D  mov         ecx,dword ptr [rsi+84h]  
00007FF66E5F9E53  mov         dword ptr [rsi+1E0h],eax  
00007FF66E5F9E59  add         ecx,dword ptr [rdi+18h]  
00007FF66E5F9E5C  mov         dword ptr [rsi+84h],ecx  
00007FF66E5F9E62  add         eax,dword ptr [rdi+18h]  
00007FF66E5F9E65  mov         dword ptr [rsi+80h],eax  
00007FF66E5F9E6B  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+499h (07FF66E5F9F59h)  
00007FF66E5F9E70  mov         r15,rax  
00007FF66E5F9E73  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+3F5h (07FF66E5F9EB5h)  
00007FF66E5F9E75  mov         rcx,rbp  
00007FF66E5F9E78  call        webrtc::AudioFrame::mutable_data (07FF66BE7FFEEh)  
00007FF66E5F9E7D  mov         rcx,qword ptr [rsi+1F8h]  
00007FF66E5F9E84  test        rcx,rcx  
00007FF66E5F9E87  cmove       rax,rcx  
00007FF66E5F9E8B  lea         rdx,[rsp+180h]  
00007FF66E5F9E93  mov         qword ptr [rdx],rax  
00007FF66E5F9E96  mov         qword ptr [rdx+8],rcx  
00007FF66E5F9E9A  mov         rcx,rdi  
00007FF66E5F9E9D  call        webrtc::DownMixFrame (07FF66E67D7B0h)  
00007FF66E5F9EA2  mov         qword ptr [rsi+208h],1  
00007FF66E5F9EAD  lea         r15,[rsp+190h]  
00007FF66E5F9EB5  mov         eax,dword ptr [rsi+80h]  
00007FF66E5F9EBB  mov         dword ptr [rsi+1E0h],eax  
00007FF66E5F9EC1  mov         ecx,dword ptr [rdi+20h]  
00007FF66E5F9EC4  mov         dword ptr [rsi+200h],ecx  
00007FF66E5F9ECA  cmp         r12d,r14d  
00007FF66E5F9ECD  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+484h (07FF66E5F9F44h)  
00007FF66E5F9ECF  mov         rcx,rbp  
00007FF66E5F9ED2  call        webrtc::AudioFrame::mutable_data (07FF66BE7FFEEh)  
00007FF66E5F9ED7  mov         r14,rax  
00007FF66E5F9EDA  lea         r12,[rsi+88h]  
00007FF66E5F9EE1  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9EE8  mov         r13,qword ptr [rsi+208h]  
00007FF66E5F9EEF  mov         rax,qword ptr [rcx]  
00007FF66E5F9EF2  call        qword ptr [rax+8]  
00007FF66E5F9EF5  mov         r8d,dword ptr [rdi+20h]  
00007FF66E5F9EF9  mov         qword ptr [rsp+30h],r14  
00007FF66E5F9EFE  mov         qword ptr [rsp+20h],r13  
00007FF66E5F9F03  mov         qword ptr [rsp+28h],1E00h  
00007FF66E5F9F0C  mov         rcx,r12  
00007FF66E5F9F0F  mov         rdx,r15  
00007FF66E5F9F12  mov         r9d,eax  
00007FF66E5F9F15  call        webrtc::acm2::ACMResampler::Resample10Msec (07FF66E67D544h)  
00007FF66E5F9F1A  test        eax,eax  
00007FF66E5F9F1C  js          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+7DFh (07FF66E5FA29Fh)  
00007FF66E5F9F22  mov         eax,eax  
00007FF66E5F9F24  mov         qword ptr [rsi+1F8h],rax  
00007FF66E5F9F2B  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9F32  mov         rax,qword ptr [rcx]  
00007FF66E5F9F35  call        qword ptr [rax+8]  
00007FF66E5F9F38  mov         dword ptr [rsi+200h],eax  
00007FF66E5F9F3E  mov         eax,dword ptr [rsi+80h]  
00007FF66E5F9F44  add         eax,dword ptr [rsi+1F8h]  
00007FF66E5F9F4A  mov         dword ptr [rsi+80h],eax  
00007FF66E5F9F50  mov         eax,dword ptr [rdi+18h]  
00007FF66E5F9F53  add         dword ptr [rsi+84h],eax  
00007FF66E5F9F59  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9F60  mov         rax,qword ptr [rcx]  
00007FF66E5F9F63  call        qword ptr [rax+10h]  
00007FF66E5F9F66  mov         ecx,dword ptr [rbp]  
00007FF66E5F9F69  cmp         qword ptr [rbp+28h],rax  
00007FF66E5F9F6D  mov         dword ptr [rsi+8],ecx  
00007FF66E5F9F70  mov         rcx,qword ptr [rbp+18h]  
00007FF66E5F9F74  mov         qword ptr [rsi+18h],rcx  
00007FF66E5F9F78  mov         qword ptr [rsi+20h],rax  
00007FF66E5F9F7C  jne         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+4C8h (07FF66E5F9F88h)  
00007FF66E5F9F7E  mov         rcx,rbp  
00007FF66E5F9F81  call        webrtc::AudioFrame::data (07FF66BE7FF20h)  
00007FF66E5F9F86  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+4DBh (07FF66E5F9F9Bh)  
00007FF66E5F9F88  lea         r8,[rsi+28h]  
00007FF66E5F9F8C  mov         rcx,rbp  
00007FF66E5F9F8F  mov         rdx,rax  
00007FF66E5F9F92  call        webrtc::ReMixFrame (07FF66E67D814h)  
00007FF66E5F9F97  mov         rax,qword ptr [rsi+28h]  
00007FF66E5F9F9B  mov         qword ptr [rsi+10h],rax  
00007FF66E5F9F9F  lea         rcx,[rsp+190h]  
00007FF66E5F9FA7  call        webrtc::AudioEncoder::EncodedInfo::EncodedInfo (07FF66E344B90h)  
00007FF66E5F9FAC  lea         rdx,[string "Process" (07FF66EC24BC5h)]  
00007FF66E5F9FB3  mov         rcx,rsi  
00007FF66E5F9FB6  call        webrtc::`anonymous namespace'::AudioCodingModuleImpl::HaveValidEncoder (07FF66E5FA69Ch)  
00007FF66E5F9FBB  mov         ebx,0FFFFFFFFh  
00007FF66E5F9FC0  test        al,al  
00007FF66E5F9FC2  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+7CDh (07FF66E5FA28Dh)  
00007FF66E5F9FC8  cmp         byte ptr [rsi+3E39h],0  
00007FF66E5F9FCF  je          webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+518h (07FF66E5F9FD8h)  
00007FF66E5F9FD1  mov         eax,dword ptr [rsi+8]  
00007FF66E5F9FD4  mov         edi,eax  
00007FF66E5F9FD6  jmp         webrtc::`anonymous namespace'::AudioCodingModuleImpl::Add10MsData+561h (07FF66E5FA021h)  
00007FF66E5F9FD8  mov         edi,dword ptr [rsi+3E40h]  
00007FF66E5F9FDE  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9FE5  mov         rax,qword ptr [rcx]  
00007FF66E5F9FE8  call        qword ptr [rax+8]  
00007FF66E5F9FEB  movsxd      rbx,eax  
00007FF66E5F9FEE  mov         ebp,dword ptr [rsi+8]  
00007FF66E5F9FF1  sub         ebp,dword ptr [rsi+3E3Ch]  
00007FF66E5F9FF7  mov         rcx,qword ptr [rsi+1D0h]  
00007FF66E5F9FFE  mov         rax,qword ptr [rcx]  
00007FF66E5FA001  call        qword ptr [rax+18h]  
00007FF66E5FA004  movsxd      rcx,eax  

</details>

**Comment by nornagon** [comment]
Thanks for the report. It looks like this was reported as merged to m80, but there's no tag in upstream with an 80.0.x version number including this fix.

**Comment by nornagon** [comment]
Actually, it looks like this fix was in [80.0.3987.159](https://chromium.googlesource.com/chromium/src/+/refs/tags/80.0.3987.159/DEPS#1506) until [80.0.3987.163](https://chromium.googlesource.com/chromium/src/+/refs/tags/80.0.3987.163/DEPS#1506), but was then reverted in [80.0.3987.165](https://chromium.googlesource.com/chromium/src/+/refs/tags/80.0.3987.165/DEPS#1506), which seems to have possibly been a ChromeOS-only build.

I'm going to roll Electron back to 80.0.3987.163, which includes several other fixes besides:

```
* 052d3b44972e (tag: 80.0.3987.165, refs/patches/upstream-head) Incrementing VERSION to 80.0.3987.165 [chrome-release-bot]
* 599754e17c97 logs: Implement size limit for user logs [Joon Ahn]
* b032a48e4ee3 Setting version to 80.0.3987.164 [chrome-release-bot]
* 9fb54ff5ece0 Publish DEPS for 80.0.3987.158 [chrome-release-bot]
| * 3770caa190b6 (tag: 80.0.3987.163) Publish DEPS for 80.0.3987.163 [chrome-release-bot]
| * e7fbe071abe9 Incrementing VERSION to 80.0.3987.163 [chrome-release-bot]
| * fc11c43603c0 Revert "Crash when a process node is destroyed while still hosting worker nodes" [Patrick Monette]
| * 3d2784d7fa18 M80 merge: Fix themes for extensions lite for supervised users [James Cook]
| * f2c5dd613815 Incrementing VERSION to 80.0.3987.162 [chrome-release-bot]
| * 4e5a9529c020 Revert "3987: pin v8 to 9c25291e705136181ede345dabcf05fb054812af." [Srinivas Sista]
| * d84d2c896ef4 3987: [iOS]Updated "shard size" to "swarming tasks" for EG tests. [John Budorick]
| * 5714fda506da Incrementing VERSION to 80.0.3987.161 [chrome-release-bot]
| * 34364febb5c0 Android: Update Play Core [Peter Wen]
| * 1a49b1135d32 Incrementing VERSION to 80.0.3987.160 [chrome-release-bot]
| * 39588f9f46ea [Merge to M80] Worker: Stop passing creator's origin for starting a dedicated worker [Hiroki Nakagawa]
| * 84c7116cd2eb Incrementing VERSION to 80.0.3987.159 [chrome-release-bot]
| * f1c37ded70b8 M80 merge: Enable supervised users to install extensions from policy allowlist [James Cook]
| * d1d9a6eb4be6 Pepper: support GpuMemoryBuffer-based frame in MediaStream Video [Shik Chen]
| * b251e5d04acc Clear context from orphan handlers when BaseAudioContext is going away [Raymond Toy]
| * fadaf051bb01 [PM] Fix invariant tracking [Patrick Monette]
| * cb6da3a7587f Crash when a process node is destroyed while still hosting worker nodes [Patrick Monette]
| * 4c737aabcb3a Protect AutofillPopupControllerImpl from being destroyed in Show(). [Vasilii Sukhanov]
| * 684baa766f80 Protect automatic pull handlers with Mutex [Hongchan Choi]
|/
* af496874d27c Incrementing VERSION to 80.0.3987.158 [chrome-release-bot]
```

**Comment by chevonc** [comment]
Ahh, interesting. Thanks for checking into this - will happily test the new build once it's ready

**Comment by TatharNuar** [comment]
I will be assisting @chevonc in testing the new build.

**Comment by nornagon** [comment]
Resolved by #25499.
